spec/models/user_spec.rb

```diff
@@ -4,4 +4,16 @@
 
 RSpec.describe User, type: :model do
   it_behaves_like 'a gds-sso user class'
+
+  describe '#resource_manager?' do
+    it 'returns false' do
+      user = build(:user)
+      expect(user.resource_manager?).to be false
```

Comment:
Maybe we should use the predicate matchers instead so these would read like `expect(user).to be_resource_manager` wdyt?